src/notation/qml/MuseScore/NotationScene/internal/EditStyle/ComboBoxDropdown.qml

```diff
@@ -102,7 +102,18 @@ ComboBox {
         }
 
         Component.onCompleted: function() {
+            if (!styleItem) {
+                return;
+            }
+
             currentIndex = indexOfValue(styleItem.value)
```

Comment:
Can't we just make a binding?

```
currentIndex: styleItem ? indexOfValue(styleItem.value) : -1
```
(And nothing in onCompleted)

Reply:
I tried this first time round, but nothing was selected when opening the dialog. Leaving `currentIndex = indexOfValue(styleItem.value)` in with the binding broke the binding completely